docs/upgrading-neutrino.md

```diff
@@ -1,202 +1,23 @@
-implementation of async/await is performance (https://github.com/MatAtBread/nodent#performance) - it's 3-4 times faster
-in a browser/node, and as much as 10 times faster on a mobile browsers, mainly due to avoiding generators
-(and therefore regenerator).
-
-If you wish to disable the fast-async polyfill, you can specify it to the Web, React, and Node.js presets from
-your `.neutrinorc.js`:
-
-```js
-module.exports = {
-  use: [
-    ['neutrino-preset-web', {
-      polyfills: { async: false }
-    }]
-  ]
-};
-```
-
-- The React preset is now additionally Babel-compiled with `transform-class-properties`, and during `development` also
-includes `transform-es2015-classes` to work around https://github.com/gaearon/react-hot-loader/issues/313.
-
-- The Node.js preset, start-server middleware, and babel-preset-env in any middleware can be debugged with the `--debug`
-CLI flag or the Neutrino option `debug`.
-
-- The Node.js preset now compiles Babel to support Node.js LTS which has moved to v6.10 from v6.9.
-
-- Babel compiles using a `cacheDirectory` to improve subsequent build times.
-
-- The Neutrino API now exposes generic `prerun` and `run` events which are triggered prior to at any command
-running, e.g. running `neutrino build` will trigger a `prebuild`, `prerun`, `build`, and `run` events.
-
-- The ESLint middleware now defaults to a `development` NODE_ENV if none is specified.
-
-- Configuration can now more easily be overridden based on the values of environment variables from `.neutrinorc.js`
-using the `env` property. Each property within `env` maps to an environment variable with key-values mapping to 
-environment values which contain further middleware. This works for any environment variable, not just `NODE_ENV`.
-For example, if you wanted to include additional middleware when `NODE_ENV` is `production`:
-
-```js
-module.exports = {
-  env: {
-    NODE_ENV: {
-      production: {
-        use: ['neutrino-middleware-pwa']
-      }
-    }
-  }
-};
-```
+# Upgrading from Neutrino v6 to v7
+
+Neutrino v7 introduces some changes, most of which are from major version bumps of dependent packages.
+To upgrade from Neutrino v6 to v7, be sure to check this list for tasks you may need to perform to use
+this latest version:
+
+- **BREAKING CHANGE** webpack-chain upgraded to v4. The breakage from this upgrade is the `.noParse` method
+which is now a plain shortcut method, allowing you to set any value.
+- **BREAKING CHANGE** Webpack upgraded to v3. While most users should need no changes for this upgrade, be
+aware that it may still be possible for something to change during this upgrade.
+- **BREAKING CHANGE** ESLint has been upgraded to v4, which may bring linting configuration
+changes.
+- **BREAKING CHANGE** The Neutrino `static` option has been removed. The Web, React, and Node.js presets will
+still copy files from a `src/static` directory if it exists, but you can override your own
```

Comment:
I think copying from `src/static` is fine by default, but any overrides of copy would obviously negate that. What wasn't sustainable was providing an option for the static directory, since it just wasn't powerful enough or didn't operate as expected.